src/frontend/src/pages/HomePage/components/GuestOrganizationInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from '@mui/material';
+import { Grid } from '@mui/material';
+import { useCurrentOrganization } from '../../../hooks/organizations.hooks';
+import React from 'react';
+import { NERButton } from '../../../components/NERButton';
+import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
+
+interface GuestOrganizationInfoButtonProps {
+  href?: string;
+  buttonText: string;
+}
+
+const NERGuestButton: React.FC<GuestOrganizationInfoButtonProps> = ({ href, buttonText }) => {
```

Comment:
The button should also take in an icon name , so that not all the icons are just the 'i'